addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -49,6 +49,12 @@ extends Resource
 ## Skip texture path that gets applied to caulk and nodrawnonsolid shaders.
 @export var skip_texture: String = "textures/special/skip"
 
+## Optional variables to include in the build profile.
+@export var build_xml_variables: Array[Array]
+
+## Optional commands to include in the build profile.
+@export var build_xml_commands: Array[Array]
```

Comment:
Looking at how build options and commands are constructed I think it'd make sense to also make these Dictionaries, but they'd be treated as `Dictionary<String><Array>`, as there can be multiple commands in a build option. Would make them easier to parse, more intuitive.